src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -13,6 +13,9 @@ import { useVenueUserStatuses } from "hooks/useVenueUserStatuses";
 import { useVenueId } from "hooks/useVenueId";
 
 import "./UserAvatar.scss";
+import { useImage } from "hooks/useImage";
+import { useState } from "react";
+import { useEffect } from "react";
```

Comment:
React imports should be always first and inline. However, after my other comment, these will probably go away.

Reply:
fixed 0d6d78b